tests/network/vmi_multus.go

```diff
@@ -287,8 +308,8 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 				err = console.SafeExpectBatch(detachedVMI, []expect.Batcher{
 					&expect.BSnd{S: "\n"},
 					&expect.BExp{R: console.PromptExpression},
-					&expect.BSnd{S: "ip link show | grep -c UP\n"},
-					&expect.BExp{R: "2"},
+					&expect.BSnd{S: "ip link show | grep -v lo | grep -c UP\n"},
```

Comment:
You sneaked in this one. Not sure it is that safe. :thinking:

Reply:
It is necessary since `lo` is down in the alpine image.

Reply:
Are we going to be able to replace this checking GA instead of using console ?

Reply:
I will try a PR